### DISCUSSION TITLE
Add "delete (read) feeds after xy time"

### DISCUSSION THREAD

**Discussion by HarriBuh** [jocmp/capyreader#192] [Feature Requests]
I just scrolled through the "old" and already read feeds and there doesn't seem to be a limit to them, is that correct?
In this case, wouldn't the app get overloaded with redundant feeds at some time (= taking more and more space on the smartphone)?

My request would be not only to limit the catched feeds by time (1,2,3 days past..), but also to automatically delete them after a certain time (days, weeks..).

Is this possible?

Cheers.

Could already be tackled with #185

**Comment by jocmp** [comment]
> there doesn't seem to be a limit to them, is that correct?

that's correct. right now there's not a limit to how many feeds are stored on device. Most articles don't take up a lot of space. Images do, but they're stored in a cache that deletes itself over time.

I definitely want to add this as a feature. I think the initial cliff would be something like 3 months, and then maybe an advanced feature to choose weeks, etc. How does that sound?

**Comment by serrq** [comment]
My current "retain feeds" value is just the latest 50 feeds in Feeder.

![1000115836](https://github.com/user-attachments/assets/1b2057a1-d11d-4bdb-90b7-6bced3daf7c6)

**Comment by jocmp** [comment]
This has been added in [2024.08.1026](https://github.com/jocmp/capyreader/releases/tag/2024.08.1026-dev). Articles will be deleted on refresh if they are 

- Read
- Not starred
- Published at least 3 months ago
